examples/dao/setup_sigma_dao.sql

```diff
@@ -4,7 +4,8 @@
 
 -- create a sigma_daos table if does not exit already.
 CREATE TABLE IF NOT EXISTS sigma_daos (
-  app_id bigint PRIMARY KEY, -- application id
+  id SERIAL PRIMARY KEY, -- auto increment id
+  app_id bigint, -- application id
```

Comment:
we need to have an index by `app_id` as well.

Reply:
Pinged you. Please check.

Reply:
provided example in a comment below.

Reply:
Tested.